lib/runner/cli/nightwatch.conf.ejs

```diff
@@ -253,6 +253,61 @@ module.exports = {
       }
     },
     //////////////////////////////////////////////////////////////////////////////////
+    // Configuration for when using the SauceLabs cloud service                      |
+    //                                                                               |
+    // Please set the username and access key by setting the environment variables:  |
+    // - SAUCE_USERNAME                                                              |
+    // - SAUCE_ACCESS_KEY                                                            |
+    //////////////////////////////////////////////////////////////////////////////////
+    sauce: {
+      selenium: {
+        host: 'ondemand.saucelabs.com',
+        port: 443
+      },
+      // More info on configuring capabilities can be found on:
+      // https://wiki.saucelabs.com/display/DOCS/Test+Configuration+Options
+      desiredCapabilities: {
+        'sauce:options' : {
+          username: '${SAUCE_USERNAME}',
+          access_key: '${SAUCE_ACCESS_KEY}',
+          // https://docs.saucelabs.com/dev/cli/sauce-connect-proxy/#--region
+          sauce_region: 'us-west-1'
+          // https://docs.saucelabs.com/dev/test-configuration-options/#tunnelidentifier
+          // parentTunnel: '',
+          // tunnelIdentifier: '',
+        }
+      },
+      disable_error_log: false,
+      webdriver: {
+        start_process: false
+      }
+    },
+    'sauce.chrome': {
+      extends: 'sauce',
+      desiredCapabilities: {
+        browserName: 'chrome',
+        screenResolution: '1280x1024',
+        browserVersion: 'latest',
+        javascriptEnabled: true,
+        acceptSslCerts: true,
+        timeZone: 'New York',
+        chromeOptions : {
+          w3c: true
+        }
+      }
+    },
+    'sauce.firefox': {
+      extends: 'sauce',
+      desiredCapabilities: {
+        browserName: 'firefox',
+        screenResolution: '1280x1024',
+        browserVersion: 'latest',
+        javascriptEnabled: true,
+        acceptSslCerts: true,
+        timeZone: 'New York'
```

Comment:
is the timeZone needed? we probably should leave it to the default value.

Reply:
Yes, it is needed, without it really weird things tend to happen, especially with timestamps on pages under test, when you are running a test in one time zone and connected to their cloud farm in another zone.

Reply:
Yeah, let's do that.

Reply:
Set to London!